packages/amplify-provider-awscloudformation/resources/rootStackTemplate.json

```diff
@@ -21,7 +21,22 @@
       "Type": "AWS::S3::Bucket",
       "DeletionPolicy": "Retain",
       "Properties": {
-        "BucketName": { "Ref": "DeploymentBucketName" }
+        "BucketName": {
+          "Ref": "DeploymentBucketName"
+        },
+        "VersioningConfiguration": {
+          "Status": "Enabled"
+        },
+        "LifecycleConfiguration": {
```

Comment:
This is the root-stack which is generated for all inited CFN stacks. We shouldn't be adding Container specific resources or rules tied to this in the root-stack if the switch if turned off my the customers. Why not move the "LifecycleConfiguration" property to the API nested stack?